docs/topics/modules/approving_your_application.adoc

```diff
@@ -0,0 +1,14 @@
+[id="approving_your_application"]
+= Approving your application
+
+If the quickstart application in the _Stage_ environment runs as expected, approve your application and promote it to the _Run_ environment:
+
+. Return to the {osio} tab which displays the *Pipeline* view and click btn:[Input Required] at the *Approve* stage of the pipeline.
+
+. Click btn:[Promote] to promote the build from the _Stage_ environment to the _Run_ environment. The roll out process from _Stage_ to _Run_ requires a few minutes.
++
+image::promote.png[Promote build]
+. Optionally, as your build is promoted to _Run_, click *helloworldvertx* or *Build #1* to view the detailed progress of the pipeline or build in your OpenShift Online console, respectively.
```

Comment:
This module is called approving your application but it talks about promoting your build more. Perhaps the name should be reassessed based on the content?

Reply:
AFAIU, we promote an application to production (in this case run) we do not promote a build. So it's either Promote your application or Approve your application. I picked the latter. But you are right, the word 'build' in a number of places needs to be replaced with application and could have created the perception. Taken care of now. Thanks for noticing.